### PR TITLE
[master] guest get user direct from hypervisor for onboarding

### PR DESCRIPTION
**Pull request by trenxiaof**
___________________________________ summary ____________________________________
ERROR:   pep8: commands failed
  py27: commands succeeded
  py36: commands succeeded
  docs: commands succeeded

pep8 error not related to this commit:
21392] /root/repo/python-zvm-sdk$ /root/repo/python-zvm-sdk/.tox/pep8/bin/flake8
./get-pip.py:90:5: E306 expected 1 blank line before a nested definition, found 0
./env/bin/activate_this.py:6:80: E501 line too long (97 > 79 characters)
./env/bin/activate_this.py:15:80: E501 line too long (95 > 79 characters)
./env/bin/activate_this.py:18:80: E501 line too long (103 > 79 characters)
./env/bin/activate_this.py:21:80: E501 line too long (94 > 79 characters)
./env/bin/activate_this.py:26:80: E501 line too long (95 > 79 characters)
./env/lib64/python2.7/os.py:24:1: E265 block comment should start with '# '

### CODE REVIEW

**Comment by trenxiaof** [comment]
@jichenjc @bjhuangr @SeanHQF please help review

File: doc/source/restapi.rst [hunk 1]
```diff
@@ -579,6 +579,35 @@ Get running information of guest.
    :language: javascript
 
 
+Get Guest user direct
+---------------------
+
+**GET /guests/{userid}/user_direct**
+
+Get the user direct of the given userid from hypervisor.
```
Comment (SeanHQF) [inline doc/source/restapi.rst]: user direct-->user directory info?

Comment (trenxiaof) [reply doc/source/restapi.rst]: use direct found in many place of zcc comment. but sure， user directory info better. will update

**Comment by SeanHQF** [comment]
@trenxiaof per @jichenjc 's comment, please change the password in the fist line of directory info to be "****" (the field after HQF0006E).
USER HQF0006E **LBYONLY** 4096m 64G G

**Comment by trenxiaof** [comment]
> @trenxiaof per @jichenjc 's comment, please change the password in the fist line of directory info to be "****" (the field after HQF0006E).
> USER HQF0006E **LBYONLY** 4096m 64G G

have updated in the code. please help review.

**Comment by SeanHQF** [comment]
@trenxiaof pls solve the conflict.

**Comment by trenxiaof** [comment]
have rebased to the latest master branch, please help merge if the commits are ok.

**Comment by SeanHQF** [comment]
lgtm

**Comment by SeanHQF** [comment]
@jichenjc pls help to merge it if no further comments, thanks!

**Pull request merged**